mods/cnc/maps/nod08b/nod08b-AI.lua

```diff
@@ -158,30 +158,30 @@ Trigger.OnAllKilledOrCaptured(GDIBase, function()
 	Utils.Do(GDI.GetGroundAttackers(), IdleHunt)
 end)
 
-Trigger.OnKilled(GDIProc, function(building)
+Trigger.OnKilled(GDIProc, function()
```

Comment:
LUA is flexible in the type of function that you pass?

I will also not lead to a memory leak of the building argument. I guess it's reference counted and it won't.

Reply:
Yes, you don't need to specify the parameters if you don't use them.

Reply:
The IDE showed it as greyed out, but didn't issue a warning.